awstransfer-s3-sam/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "Selective file transfer between SFTP server and S3 using AWS Transfer Family",
+  "description": "This pattern shows how to use AWS Transfer Family to list and transfer specific files between an SFTP server and Amazon S3 bucket.",
+  "language": "YAML",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "The remote SFTP server is simulated using AWS Transfer Family SFTP Server for this pattern. In real use case, this can be any remove SFTP server outside of AWS.",
```

Comment:
```suggestion
      "The remote SFTP server is simulated using AWS Transfer Family SFTP Server for this pattern. In a real use case, this can be any remote SFTP server.",
```

Reply:
Fixed.